examples/wifi-echo/server/esp32/main/af-main.h

```diff
@@ -0,0 +1,162 @@
+/**
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *
+ *    Copyright (c) 2020 Silicon Labs
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+/***************************************************************************/
+/**
+ * @file
+ * @brief
+ *******************************************************************************
+ ******************************************************************************/
+
+#ifndef SILABS_AF_MAIN_H
+#define SILABS_AF_MAIN_H
+
+#include "af-types.h"
+
+//#include CONFIGURATION_HEADER
```

Comment:
Remove commented out code?

This commented out include seems to occur in other places as well.

Reply:
As I mentioned above, the goal is to land something with as few differences from the original SiLabs code as possible, subject to the "it compiles and runs" constraint.  Once this lands, we will definitely be doing a bunch of cleanup, which will include removing the commented-out bits here.

Reply:
I filed https://github.com/project-chip/connectedhomeip/issues/1611 on this.